src/domain/submission/services/exporter/meca-exporter.ts

```diff
@@ -1,37 +1,30 @@
 import { SubmissionId } from '../../types';
 import { SubmissionExporter } from './types';
+// import { FileService } from '../../../file/services/file-service';
 
 export class MecaExporter implements SubmissionExporter {
-    export(id: SubmissionId): Promise<Buffer> {
-        return Promise.resolve(new Buffer(`this-is-a-test ${id}`));
-        /*
-        @todo: Implement something like...
+    // constructor(private readonly fileService: FileService) {}
 
-        const uploadedFiles = manuscript.files.map((file, index) => ({
-          id: file.id,
-          filename: file.filename,
-          content: getContent(file),
-          type: file.type,
-          mimeType: file.mimeType,
-          index,
-        }))
+    async export(submissionId: SubmissionId): Promise<Buffer> {
+        return Promise.resolve(new Buffer(`this-is-a-test ${submissionId}`));
+        // const manuscriptFile = await this.fileService.findManuscriptFile(submissionId);
```

Comment:
Is this to be re-enabled code later?

Reply:
yes this will be in follow up ticket #706